docs/Project.toml

```diff
@@ -1,5 +1,4 @@
 [deps]
-AbstractAlgebra = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
```

Comment:
Can we keep this? So that I can build the docs locally without introducing a diff:
```
cd docs
julia --color=yes --project -e 'using Pkg ; Pkg.develop(path="..")'
julia --color=yes --project make.jl
```
Or how else am I supposed to build them?

Reply:
May be stacking on top of `src/Project.toml`?

If there is no problem with this, I would also like to keep it.